flow-report/test/App-test.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {App} from '../App';
+import {render} from '@testing-library/preact';
+
+const flow = require('../../lighthouse-core/test/fixtures/fraggle-rock/reports/sample-lhrs.json');
```

Comment:
please use fs / LH_ROOT here to ease the upcoming transition to esmoudles, which can't import JSON.

Reply:
also, are we going with `flow` or `flowReport` or `flhr` or ?

Reply:
`flowReport`

Reply:
> please use fs / LH_ROOT here to ease the upcoming transition to esmoudles, which can't import JSON.

If it's going to be `tsx`, should they just start as esmodules? OTOH, since these files are going to be `tsx` and need a build step to run, does it matter?